package.json

```diff
@@ -170,6 +172,7 @@
             ".vbs": "cscript //Nologo",
             ".scala": "scala",
             ".jl": "julia",
+            ".asm": "nasm -f elf64 $fileName && ld -s -o $fileNameWithoutExt $fileNameWithoutExt.o && ./$fileNameWithoutExt",
```

Comment:
- may need to 'cd $dir' first
- Did you test it on Win/macOS/Lunix?

May have some minor update:
```
".asm": "cd $dir && nasm -f elf64 $fileName && ld -s -o $fileNameWithoutExt $fileNameWithoutExt.o && $dir$fileNameWithoutExt"
```